app/controllers/AddApplicationStart.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import config.{ApplicationConfig, ErrorHandler}
+import javax.inject.{Inject, Singleton}
+import play.api.i18n.MessagesApi
+import service.{AuditService, SessionService}
+
+import scala.concurrent.Future
+
+@Singleton
+class AddApplicationStart @Inject()(val auditService: AuditService,
```

Comment:
Just a thought but if this controller was renamed to something along the lines of AddApplication then it could contain all the endpoints for adding an application rather than just the start page.